README.md

```diff
@@ -152,9 +152,16 @@ $ python3 tests/test_client.py -i atmediamanager -s atmediamanager -e qa --asset
 To run unit tests:
```

Comment:
Couldn't comment directly on the lines (seem to be outside of the range which Github code review allows - 144-150), but if `tests/test_client.py` and the older tests are deleted, we should probably remove the reference to those end-to-end tests